### ISSUE TITLE
onTransportClose event is never handled

### ISSUE THREAD

**Issue by kasperdun** [crow-misia/libmediasoup-android#18]
Hi, 
event onTransportClose have to be handled when call sendTransport.close()
But when I call sendTransport.close() event is not handled

_Also this is not work in your demo_

```kt
sendTransport.produce(
                listener = object : Producer.Listener {
                    override fun onTransportClose(producer: Producer) {
                        Log.d(TAG) { "onTransportClose()" } // never printed
                    }
                },
                track = track,
                encodings = emptyList(),
                codecOptions = null,
                appData = null,
            )
```

**Comment by crow-misia** [comment]
If Producer is closed before Transport, it will not be notified.

Listener may not be handling it properly.
Can you check if other callback methods of the Producer are called?

**Comment by kasperdun** [comment]
I know it and I checked, when I call sendTransport.close() 
producer is not closed at this moment

**Comment by crow-misia** [comment]
Fixed in 0.14.1.
Method was not called because of an error in object type conversion.
